### ISSUE TITLE
Items not linking from Omeka

### ISSUE THREAD

**Issue by akthomason** [performant-software/Neatline#390]
When I go to add an item to a record, nothing shows up. I have plenty of items in both collections and exhibits within Omeka, but the linking from Omeka to neatline does not seem to be working. We only have the base neatline installation at the moment. The search box says "searching" and briefly in the lower right hand corner it will say "loading" but nothing ever loads. Neatline seems to be working fine as far as creating a new exhibit. Any help appreciated.

**Comment by rondauva** [comment]
To facilitate testing, which versions of Omeka and Neatline are you using? Thanks!

**Comment by akthomason** [comment]
Omeka 2.4 and Neatline 2.5.1 Thanks!

**Comment by davidmcclure** [comment]
Hey @akthomason,

Are the Omeka items public? If I recall correctly, I think they need to be public in order to show up in the dropdown select in Neatline.

**Comment by akthomason** [comment]
Yes, they are public, at least the ones I'm trying to link to. Thanks for the tip though! The good news is importing items from within Omeka to Neatline is working, but the import from within a Neatline map is still not working.

**Comment by davidmcclure** [comment]
Hm ok, worth a shot. Maybe try this - using Chrome, open up the Javascript console (right click on the page, click "Inspect," then switch to the "Console" tab), and then try to link a Neatline record with an item. Do you see an error on the Javascript console?

**Comment by jeremyboggs** [comment]
@akthomason Can you provide some more detail about this issue? Are you still running into this issue?
